dotnet/src/webdriver/SeleniumManager.cs

```diff
@@ -71,62 +71,34 @@ static SeleniumManager()
         }
 
         /// <summary>
-        /// Determines the location of the correct driver.
+        /// Determines the location of the browser and driver binaries.
         /// </summary>
-        /// <param name="options">The correct path depends on which options are being used.</param>
+        /// <param name="arguments">List of arguments to use when invoking Selenium Manager.</param>
         /// <returns>
-        /// The location of the driver.
+        /// An array with two entries, one for the driver path, and another one for the browser path.
         /// </returns>
-        public static string DriverPath(DriverOptions options)
+        public static Dictionary<string, string> BinaryPaths(string arguments)
```

Comment:
It is public API, where we should be careful. Can we split this method to two methods? For `BrowserBinaryPath` and `DriverBinaryPath`?

Dictionary in public API doesn't look intuitive for users.

Reply:
Theoretically, `BinaryPaths` should only be used from `DriverFinder`. Should we make it private?